### PR TITLE
doc(LICENSE) MIT :page_facing_up:

### PR DESCRIPTION
**Pull request by 6ix4**

### CODE REVIEW

**Comment by 6ix4** [comment]
ฟ

**Review by 6ix4 — COMMENTED** [review]
GO

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
```
Comment (6ix4) [inline LICENSE:20]: good License

**Review by ThTiny — APPROVED** [review]
💖

**Pull request merged**